src/window.rs

```diff
@@ -560,18 +560,80 @@ pub struct Window<T: Layout> {
 #[derive(Debug, Copy, Clone)]
 pub struct AnimationState { }
 
+pub struct ScrollStates(pub FastHashMap<ExternalScrollId, ScrollState>);
```

Comment:
`ScrollStates` should be `pub(crate)` for now, so that nobody can mess with the scroll state programatically (for now, anyways, scrolling programmatically should change in the future).

Reply:
Agreed; done as requested!